javascript/javascript3/week3/shoppingCart.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Users repositories</title>
+
+</head>
+
+<body>
+
+
+
+    <script>
+        class Product {
+            constructor(name, price) {
+                this.name = name;
+                this.price = price;
+            }
+
+            convertToCurrency(currency) {
+                const exchangeCurrency = currency.toUpperCase();
+                fetch(
+                        `https://free.currconv.com/api/v7/convert?q=DKK_${exchangeCurrency}&compact=ultra&apiKey=887a84c9cd061cfbd5fa`
+                    )
+                    .then((response) => response.json())
+                    .then((data) => {
+                        const exchangeRate = Object.values(data)[0];
+                        const exchangeValue = (this.price * exchangeRate).toFixed(2) + currency;
+                        console.log(exchangeValue);
+                    })
+                    .catch((error) => {
+                        throw error;
```

Comment:
Here I believe that it's better just to log the error `console.log(error)` then throwing it. In general extra care has to be taken into consideration when using `throw`